components/post_view/post_header/post_header.tsx

```diff
@@ -167,12 +196,43 @@ export default class PostHeader extends React.PureComponent<Props> {
             colon = (<strong className='colon'>{':'}</strong>);
         }
 
+        const userCustomStatus = this.props.customStatus;
+        const isCustomStatusSet = userCustomStatus && userCustomStatus.emoji;
+        const isCurrentUser = this.props.post.user_id && this.props.post.user_id === this.props.userID
+        if (!isSystemMessage && isCustomStatusSet) {
+            customStatus = (
+                <CustomStatusEmoji
+                    userID={this.props.post.user_id}
+                    showTooltip={true}
+                    emojiSize={14}
+                    emojiStyle={{
+                        margin: '4px 0px 0px 4px',
```

Comment:
nit - 
```suggestion
                        margin: '4px 0 0 4px',
```

Also, check if shorthand `0 4px` works